src/components/ProgressBar/ProgressBar.js

```diff
@@ -1 +1,6 @@
-export { ProgressBar as default } from 'react-bootstrap';
+import { ProgressBar as OriginalProgressBar } from 'react-bootstrap';
+
+const ProgressBar = OriginalProgressBar;
+delete ProgressBar.propTypes.children;
```

Comment:
it seems a bit dangerous to delete `propTypes.children` on all ProgressBars? What does this do?

Reply:
Well to make Stacked progress bar that I needed I wrote it like this.
``` html
<ProgressBar>
  <ProgressBar />
  <ProgressBar />
</ProgressBar>
```
and `ProgressBar` doesn't allow anything inside but `ProgressBar`. So it was quick fix to allow tooltips that should go away later. I will keep it in mind. Thanks for feedback 😄